evac/worker.py

```diff
@@ -1042,8 +1044,7 @@ def psql_error(self):
 
 if __name__ == "__main__":
     w = Worker()
-    try:
```

Comment:
you can not change this

Reply:
its easier to debug when there is no try catch in evac/worker.py you can see when exception happens on stderr